### PR TITLE
MES-5647 CPC Office Page Terminate

### PR DESCRIPTION
**Pull request by tomcrawleyy**
## Description
Delegated CPC Office Page Terminate Journey
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/pages/office/cat-cpc/office.cat-cpc.page.html [hunk 1]
```diff
@@ -33,36 +43,59 @@ <h4>Test summary</h4>
...
+              [display]="isDelegated"
+              [isDelegated]="isDelegated"
+              [formGroup]="form"
+              [debriefWitnessed]="pageState.debriefWitnessed$ | async"
+              [outcome]="pageState.testOutcome$ | async"
+              (debriefWitnessedChange)="debriefWitnessedChanged($event)">
+            </debrief-witnessed>
+
+            <candidate-description
+              *ngIf="!(isDelegated && !isPass())"
```
Comment (andrewsetterfield) [inline src/pages/office/cat-cpc/office.cat-cpc.page.html]: is !isPass here correct? doesnt look like its on any of the designs for any of the outcomes (pass, fail, term)

File: src/pages/office/cat-cpc/office.cat-cpc.page.html [hunk 2]
```diff
@@ -33,36 +43,59 @@ <h4>Test summary</h4>
...
+              [outcome]="pageState.testOutcome$ | async"
+              (candidateDescriptionChange)="candidateDescriptionChanged($event)">
             </candidate-description>
 
-            <identification [display]="pageState.displayIdentification$ | async" [formGroup]="form"
-                            [identification]="pageState.identification$ | async"
-                            [outcome]="pageState.testOutcome$ | async"
-                            (identificationChange)="identificationChanged($event)">
+            <identification
+              *ngIf="!(isDelegated && !isPass())"
```
Comment (andrewsetterfield) [inline src/pages/office/cat-cpc/office.cat-cpc.page.html]: is !isPass here correct? doesnt look like its on any of the designs for any of the outcomes (pass, fail, term)

File: src/pages/office/cat-cpc/office.cat-cpc.page.html (hunk 1 above)
Comment (tomcrawleyy) [reply src/pages/office/cat-cpc/office.cat-cpc.page.html]: you're right, at this stage it should only be isPass, if its not meant to be on the pass journey that will have to be amended again when we do the pass journey story, pushing a change now

File: src/pages/office/cat-cpc/office.cat-cpc.page.html [hunk 3]
```diff
@@ -33,36 +43,59 @@ <h4>Test summary</h4>
...
+            </combination>
+
+            <assessment-report *ngIf="isFail() && !isDelegated"
                                [assessmentReport]="pageState.assessmentReport$ | async"
                                [formGroup]="form"
                                (assessmentReportChange)="assessmentReportChanged($event)">
             </assessment-report>
 
-            <cpc-debrief-card *ngIf="displayIfFail(outcome) && pageState.combination$ | async"
+            <cpc-debrief-card *ngIf="isFail() && pageState.combination$ | async"
```
Comment (andrewsetterfield) [inline src/pages/office/cat-cpc/office.cat-cpc.page.html]: looks like debrief witnessed is needed on all 3 outcomes (from miro), wont isFail cause it to be hidden

File: src/pages/office/cat-cpc/office.cat-cpc.page.html (hunk 1 above)
Comment (andrewsetterfield) [reply src/pages/office/cat-cpc/office.cat-cpc.page.html]: looks like candidate description is there for all 3 non delEx outcomes, and not at all for delEx.  Im only going off the miro screens tho.

File: src/pages/office/cat-cpc/office.cat-cpc.page.html (hunk 3 above)
Comment (tomcrawleyy) [reply src/pages/office/cat-cpc/office.cat-cpc.page.html]: this isn't debrief witnessed and i didnt change the logic here just removed the necessity for a parameter and changed the name of it to something more appropriate

Comment (tomcrawleyy) [reply src/pages/office/cat-cpc/office.cat-cpc.page.html]: besides this is from the rebase against fail journey, not going to be part of this pr when it gets merged in

**Review by andrewsetterfield — APPROVED** [review]

**Review by szabikr — APPROVED** [review]

**Pull request merged**